doc/content/source/filters/AzimuthalAngleFilter.md

```diff
@@ -0,0 +1,27 @@
+# AzimuthalAngleFilter
+
+!alert note
+`AzimuthalAngleFilter` can only be added to problems when the input files contains a [OpenMCCellAverageProblem](OpenMCCellAverageProblem.md)
+in the `[Problem]` block. Otherwise, attempting to add this filter will result in an error.
+
+## Description
+
+The `AzimuthalAngleFilter` object provides a thin wrapper around an OpenMC `AzimuthalFilter` which local tallies (added through the
```

Comment:
In the interest of technical maintenance, do we want to link to the OpenMC documentation instead of re-writing it? In case something changes in the future, this could slip through the cracks (same comment applies for other filters)